### ISSUE TITLE
Contribution Helpers

### ISSUE THREAD

**Issue by pastaq** [SteamDeckHomebrew/decky-loader#132]
## The problem

Currently it is very difficult to test and validate contributions to decky-loader as there is not a documented pipeline for building the executable outside using the workflows from the official GitHub repository. Forking the repo doesn't duplicate these workflows either leading to the need to submit a PR before testing can be done with a real-world scenario. This slows the development process as the contributor must wait on workflow approval before testing each change and distracts the core dev team from other issues with low priority troubleshooting on unfinished code.

## Solutions
In order to identify a specific resolution the exact parameters of the build process and all dependencies will need to be identified. Once available, a bash makefile that replicates the workflow locally can be produced. This will ensure code can be validated before a PR is submitted and should expidite the process.
